CHANGELOG.md

```diff
@@ -16,6 +17,11 @@ All notable changes to this project will be documented in this file. The format
 
 ## [Unreleased]
 
+## [1.1.103] - 2024-08-13
```

Comment:
I think this needs to be

```
## [1.2.0] - 2024-08-13
```

Reply:
Changed

Reply:
Yes, I asked @arkadiuszos4chain to fix this with his update